bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.html

```diff
@@ -1,7 +1,9 @@
 <div class="ktb-trigger-sequence pl-3 pr-3 pb-3">
   <div fxFlex="66">
     <div fxLayout="column" *ngIf="state === 'ENTRY'">
-      <h2 uitestid="keptn-trigger-entry-h2">Trigger a new sequence for project {{ projectName }}</h2>
+      <div fxLayout="row" fxFlexAlign="space-between">
```

Comment:
What's the reason for adding a fxLayout here? I don't see any difference

Reply:
You are right, it is a leftover from one layout iteration I tried. Thx for noticing.